java-frontend/src/test/java/org/sonar/java/se/ExplodedGraphWalkerTest.java

```diff
@@ -440,6 +441,19 @@ public void init(MethodTree methodTree, CFG cfg) {
     assertThat(visitedMethods).containsExactly("test", "privateMethod");
   }
 
+  @Test
+  public void constraints_on_field_reset() {
+    List<File> classpath = Collections.singletonList(new File("target/test-classes"));
```

Comment:
`target/test-classes` is by default on the classpath, see https://github.com/SonarSource/sonar-java/blob/c08b6132063c570f76342f06204638ba895240c2/java-frontend/src/test/java/org/sonar/java/se/JavaCheckVerifier.java#L182  , so this is not necessary